src/main/java/de/unistuttgart/iste/gits/media_service/dapr/TopicPublisher.java

```diff
@@ -0,0 +1,54 @@
+package de.unistuttgart.iste.gits.media_service.dapr;
+
+import de.unistuttgart.iste.gits.common.dapr.CrudOperation;
+import de.unistuttgart.iste.gits.common.dapr.ResourceUpdateDTO;
+import de.unistuttgart.iste.gits.media_service.persistence.dao.MediaRecordEntity;
+import io.dapr.client.DaprClient;
+import io.dapr.client.DaprClientBuilder;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+/**
+ * Component that takes care of publishing messages to a dapr Topic
+ */
+@Component
+@Slf4j
+public class TopicPublisher {
```

Comment:
Stupid question but will this be moved in the common code in the future? would make sense, right?